client/src/components/_common/Button/Button.module.css

```diff
@@ -0,0 +1,54 @@
+.c-button {
+  composes: c-button from '../../../styles/components/c-button.css';
+}
+.c-button:focus,
+[class*='c-button--']:focus {
+  outline: none;
+}
+
+.c-button--primary {
+  composes: c-button--primary from '../../../styles/components/c-button.css';
+  transition: 0.3s;
+}
+
+.c-button--secondary {
+  composes: c-button--secondary from '../../../styles/components/c-button.css';
+  transition: 0.3s;
+}
+.c-button--secondary:disabled {
+  color: var(--global-color-primary--light);
+  border: var(--global-color-primary--light);
+}
+.c-button--secondary:not(:disabled):active {
+  color: var(--global-color-primary--x-light);
+  background-color: var(--global-color-primary--x-dark);
+}
+
+.c-button--size-small {
+  composes: c-button--size-small from '../../../styles/components/c-button.css';
+}
+.c-button--width-short {
+  composes: c-button--width-short from '../../../styles/components/c-button.css';
+}
+.c-button--width-medium {
+  composes: c-button--width-medium from '../../../styles/components/c-button.css';
+}
+.c-button--width-large {
+  composes: c-button--width-large from '../../../styles/components/c-button.css';
+}
+
+.c-button--as-link {
+  composes: c-button--as-link from '../../../styles/components/c-button.css';
+  background-color: transparent;
+  border-color: transparent;
+}
+.c-button--as-link:hover {
+  text-decoration: underline;
+}
+
+.c-button__icon--before {
+  composes: c-button__icon--before from '../../../styles/components/c-button.css';
+}
+.c-button__icon--after {
+  composes: c-button__icon--after from '../../../styles/components/c-button.css';
+}
```

Comment:
**ℹ️  Opportunity for Brevity**

The `.c-button--` and `.c-button__` parts of the class names in this file are superfluous yet innocuous. If you wanted, you could remove them and just have `primary`, `secondary`, `size-small`, `width-…`, `as-link`, et cetera.

<details>

The `.c-button--` and `.c-button__` came from `c-button.css`, which is a global stylesheet (CSS is natively global). Those prefixes make each class unique, so another class can use `.o-something--size-small` without conflict.

The `Button.module.css` is a CSS _module_, so the class names—at build time—become globally unique because of an automatically appended a suffix, like `_43pur_17` ([screenshot](https://user-images.githubusercontent.com/62723358/153311435-578da8e6-c039-45b4-b441-94ec09096e14.png)).

</details>